### ISSUE TITLE
Feature request: include the new bucket_id in the todo_copied webhook 

### ISSUE THREAD

**Issue by ArondeParon** [basecamp/bc3-api#168]
**Problem description:**

- When moving a todo to a new bucket, the "todo_copied" webhook is fired.
- The webhook includes both the `recording.id` (old todo id) as the `details.copy_id` (new todo id) properties
- Todo's are only directly accessible in Basecamp when _both_ `todo_id` and `bucket_id` are known
- The current implementation makes it very hardto keep track of a todo when it is moved

**Proposed solution:**

Include `bucket_id` in the webhook response.

**Comment by rosa** [comment]
@ArondeParon, good point, you are quite right. I just deployed a change to include additional `copy` details for the copied recording (the new one, corresponding to the `details.copy_id` id), following the same format as the original `recording` also included in the payload. I have also added an additional example to the [webhook docs](https://github.com/basecamp/bc3-api/blob/master/sections/webhooks.md#webhooks).
